typings/typings.d.ts

```diff
@@ -0,0 +1 @@
+export declare const process : { env: { NODE_ENV: string } }
```

Comment:
extra space after `process` 🤓

Reply:
Also, I would declare this as 

```ts
export declare const process: { env: { NODE_ENV: 'development' | 'production' } };
```

so that this is a type error:

```ts
if (process.env.NODE_ENV !== 'productio') { // <-- typo
  // ...
}
```

Reply:
Type fixed!